release/long_running_tests/long_running_tests.yaml

```diff
@@ -154,7 +154,7 @@
 
 - name: serve
   cluster:
-    app_config: app_config.yaml
```

Comment:
Can we just modify the original?

Reply:
that's an option as well but since long_running_tests has its own yaml entry point now that splits its tests, maybe more explicitly to put serve only dependencies in its own place? In addition I think serve tests might also need different # of nodes based on its use of replicas that could require its own compute template too.

Reply:
it's fine to put serve deps into that yaml. this is because for long running tests, the configuration should be uniform and they are kicked off and managed the same way. 

the other serve tests should go into a release/serve directory.